src/main/java/com/bandwidth/sdk/numbers/exception/ExceptionUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.bandwidth.sdk.numbers.exception;
+
+import java.util.concurrent.CompletableFuture;
+
+public class ExceptionUtils {
```

Comment:
I see this used pattern in Messaging SDK and Number SDK (with only the Exception name being different).  Might make sense to pull this out functionality to a shared lib.